### PR TITLE
Remove (int) since value is filtered already

### PR DESCRIPTION
**Pull request by nvyush**
Pull Request for Issue # .

### Summary of Changes
Simple remove (int) since value of $Itemid is filtered already by getInt(). If set default value to 0, then ```$app->input->getInt('Itemid', 0)``` never returns other than int, so explicit type conversion is not necessary.

### Testing Instructions
Code review.

### Documentation Changes Required
Not needed.

### CODE REVIEW

File: libraries/cms/module/helper.php [hunk 1]
```diff
@@ -377,13 +377,13 @@ protected static function &load()
 	public static function getModuleList()
 	{
 		$app = JFactory::getApplication();
-		$Itemid = $app->input->getInt('Itemid');
+		$Itemid = $app->input->getInt('Itemid', 0);
```
Comment (Quy) [inline libraries/cms/module/helper.php]: Without the 2nd parameter, it defaults to `0`.
`public static function getInt($name, $default = 0, $hash = 'default')`

Comment (nvyush) [inline libraries/cms/module/helper.php]: @Quy
Without the 2nd parameter, it defaults to ```null```. So default ```0``` is required.

**Comment by RonakParmar** [comment]
I have tested this item :white_check_mark: successfully on ea044b15e6d1f3be1282ede488de7696663e737d<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/16531">issues.joomla.org/tracker/joomla-cms/16531</a>.</sub>

File: libraries/cms/module/helper.php (hunk 1 above)
Comment (mbabker) [inline libraries/cms/module/helper.php]: @Quy you're looking at the signature for `JRequest::getInt()`, different than how `JInput` handles this method (which doesn't actually exist but works through PHP's `__call()` method).  This is fine as is.

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on ea044b15e6d1f3be1282ede488de7696663e737d<br /><br />Code review.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/16531">issues.joomla.org/tracker/joomla-cms/16531</a>.</sub>

**Comment by franz-wohlkoenig** [comment]
RTC after two successful tests.

**Pull request merged**